docs/technical_reference/property_models/mc_aq_sol.rst

```diff
@@ -9,87 +9,115 @@ This MCAS property package
+   "Phase osmotic pressure", ":math:`\Pi`", "pressure_osm_phase", "[p]", ":math:`\text{Pa}`"
+   "Ion component electrical mobility", ":math:`\mu_e`", "elec_mobility_phase_comp", "[p,j]", ":math:`\text{m}^2\text{ }\text{V}^{-1}\text{ }\text{s}^{-1}`"
+   "Ion component transport number", ":math:`t`", "trans_num_phase_comp", "[p, j]", ":math:`\text{dimensionless}`"
+   "Phase equivalent conductivity", ":math:`\Lambda`", "equiv_conductivity_phase", "[p]", ":math:`\text{m}^2 \text{ } \Omega^{-1} \text{ mol}^{-1}`"
+   "Phase electrical conductivity", ":math:`\lambda`", "elec_cond_phase", "[p]", ":math:`\Omega^{-1} \text{ m}^{-1}`"
+   "Component activity coefficient", ":math:`\gamma`", "act_coeff_phase_comp", "[j]", ":math:`\text{dimensionless}`"
+   "Debye-Huckel constant A", ":math:`A`", "deby_huckel_constant", "none", ":math:`\text{dimensionless}`"
+   "Ionic Strength", ":math:`I`", "ionic_strength_molal", "none", ":math:`\text{mol kg}^{-1}`"
+   "Mass diffusivity of solute", ":math:`D`", "diffus_phase_comp", "[p, j]", ":math:`\text{m}^2 \text{ s}^{-1}`"
+
+
+
+Relationships
+-------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Component charge-equivalent molar flow rate", ":math:`\tilde{N}=N\left|z\right|`"
+   "Component charge-equivalent molar concentration", ":math:`\tilde{n}=n\left|z\right|`"
+   "Component mass fraction", ":math:`x_j=\frac{M_j}{\sum_j{M_j}}`"
+   "Mass density of aqueous phase", ":math:`\rho=1000 \text{ kg m}^{-3}` or :math:`\rho=\rho_w + \textbf{f} \left(\sum_{j\in solute}{x_j}, T\right)` :sup:`1`"
+   "Mass density of solvent water", ":math:`\rho_w=\textbf{f}\left(T\right)` :sup:`1`"
+   "Phase volumetric flowrate", ":math:`Q=\frac{\sum_j{N_j m_{Nj}}}{\rho}`"
+   "Total volumetric flowrate", ":math:`Q_{tot}=\sum_p{Q_p}`"
+   "Component molar fraction", ":math:`y_j=\frac{N_j}{\sum_j{N_j}}`"
+   "Component molality", ":math:`b=\frac{N}{N_{H_2O} m_{N\text{H_2O}}}`"
+   "Kinematic viscosity", ":math:`\nu=\mu\rho^{-1}`"
+   "Phase osmotic pressure", ":math:`\Pi=RT\sum_{j\in solute}{n_j}`"
+   "Ion component electrical mobility", ":math:`\mu_e=\frac{D\left|z\right|F}{RT}`"
+   "Ion component transport number", ":math:`t_j=\frac{\left|z_j\right|\mu_{ej} n_j}{\sum_{j\in ion}{\left|z_j\right|\mu_{ej} n_j}}`"
+   "Phase equivalent conductivity", ":math:`\Lambda=\frac{\sum_{j\in ion}{F\left|z_j\right|\mu_{ej} n_j}}{\sum_{j\in cation}{\left|z_j\right|n_j}}`"
+   "Phase electrical conductivity", ":math:`\lambda=\Lambda\sum_{j\in cation}{\left|z_j\right|n_j}`"
+   "Debye-Huckel constant", ":math:`A=\frac{\left(2 \pi N_A\right)^{0.5}}{log(10)} \left(\frac{\textbf{e}^2}{4 \pi \epsilon \epsilon_0 kT}\right)^{\frac{3}{2}}`"
+   "Ionic strength", ":math:`I=0.5\sum_{j\in ion}{z_j^2b_j}`"
+   "Component mass diffusivity", ":math:`D \text{ }[\text{m}^2 \text{ s}^{-1}]=\frac{\chi_{1}}{(\mu \text{ }[\text{cP}])^{\chi_{2}}(V \text{ }[\text{cm}^3 \text{ mol}^{-1}])^{\chi_{3}}}`"
```

Comment:
I recommend specifying this equation is applicable only under a specific method (H-L) and condition, rather than general component.  Add a reference if not already.

Reply:
@hunterbarber Do you want to add this as a note? And if so, then we should also update the reference list as Xiangyu mentioned (assuming it's not already among the references listed)

Reply:
Yes, reference list is updated in current version. I have the changes ready for the commenting, just fighting being able to push because I'm on a different machine. It should be done in a second.

Reply:
Should be addressed in the most recent commits